src/widgets/chart.rs

```diff
@@ -71,23 +141,36 @@ impl<'a> Axis<'a> {
 /// Used to determine which style of graphing to use
 #[derive(Debug, Default, Display, EnumString, Clone, Copy, Eq, PartialEq, Hash)]
 pub enum GraphType {
-    /// Draw each point
+    /// Draw each point. This is the default.
     #[default]
     Scatter,
-    /// Draw each point and lines between each point using the same marker
+    /// Draw a line between each following point.
+    ///
+    /// This means your dataset can go back and this will draw a line backward.
```

Comment:
I'm not sure this is clear enough, I want to communicate the fact that you can do this sort of thing
![image](https://github.com/ratatui-org/ratatui/assets/36198422/6f7c4391-ba4e-4d98-8388-88ab46ef3631)

Reply:
"The order of the lines will be the same as the order of the points in the dataset, which allows this widget to draw lines both left-to-right and right-to-left."

Reply:
That's way better, I'll update